src/test/java/io/kokuwa/maven/helm/PackageMojoTest.java

```diff
@@ -40,20 +45,26 @@ void dependencies(PackageMojo mojo) {
 	@Test
 	void appVersion(PackageMojo mojo) {
 		mojo.setAppVersion("0815");
+		mojo.mavenProject.setArtifact(new DefaultArtifact("io.kokuwa.maven", "helm-maven-plugin",
+				"1.0.0", "", "", "", null));
 		assertHelm(mojo, "package src/test/resources/simple --destination target/helm/repo --app-version 0815");
 	}
 
 	@DisplayName("with chartVersion stable")
 	@Test
 	void chartVersionStable(PackageMojo mojo) {
 		mojo.setChartVersion("0.0.1");
+		mojo.mavenProject.setArtifact(new DefaultArtifact("io.kokuwa.maven", "helm-maven-plugin",
+				"1.0.0", "", "", "", null));
 		assertHelm(mojo, "package src/test/resources/simple --destination target/helm/repo --version 0.0.1");
 	}
 
 	@DisplayName("with chartVersion snapshot")
 	@Test
 	void chartVersionSnapshot(PackageMojo mojo) {
 		mojo.setChartVersion("0.0.1-SNAPSHOT");
+		mojo.mavenProject.setArtifact(new DefaultArtifact("io.kokuwa.maven", "helm-maven-plugin",
+				"1.0.0", "", "", "", null));
```

Comment:
Q: not quite understand the purpose of setting this artifact for the test

Reply:
The test project that it is using here does not have an artifact, which will cause a null pointer in the package mojo when we try to assign our dummy file to an artifact that doesn't exist. So, I'm just setting one programmatically so the tests can proceed